### DISCUSSION TITLE
how to remove `via` and package version number in the prompt?

### DISCUSSION THREAD

**Discussion by hopezh** [starship/starship#5779] [Q&A]
My current `starship` prompt looks like this: `~ via  v3.10.13 via 🅒 ai`.

May I ask how to config `starship` to remove the text `via` and the `package version number` from the prompt?

Thanks.

**Comment by matoruru** [comment]
@hopezh
You should set your custom config in `~/.config/starship.toml`.
I'm not sure which component you are talking about, but

For example if it's Bun, you should set this in the config:

```toml
[bun]
format = '[🍔 $version](bold green) '
```
(The default is `'via [🍔 $version](bold green) '`)

Refer this documentation and try to overwrite the `format` in your config.
https://starship.rs/config/#configuration
